catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceQueryServiceImpl.java

```diff
@@ -150,6 +152,21 @@ public WorkspaceQueryServiceImpl(QueryUpdateSubscriber queryUpdateSubscriber,
         }
     }
 
+    public void setQueryTimeInterval(Integer queryTimeInterval) {
+        notNull(queryTimeInterval, "queryTimeInterval must be non-null");
+        if(queryTimeInterval < 0 || queryTimeInterval > 1440) {
```

Comment:
also is 0 a valid query time we want to allow? I can't say I know what a good minimum value is for this but If i had to guess i'd say at least 1 minute.